src/Core/Content/Media/Thumbnail/ThumbnailService.php

```diff
@@ -462,6 +462,10 @@ private function writeThumbnail($thumbnail, MediaEntity $media, string $url, int
             case 'image/jpeg':
                 imagejpeg($thumbnail, null, $quality);
 
+                break;
+            case 'image/webp':
+                imagewebp($thumbnail, null, $quality);
```

Comment:
Please add a functions_exists with an exception when it is missing. webp needs to be compiled

Reply:
Alright, will do.

Reply:
Added function_exists check